Update AndroidManifest.xml

Austinj0 opened the pull request:
Added needed permissions for document collection

Austinj0 committed:
Update AndroidManifest.xml

Added needed permissions for document collection

mrestuccia commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,7 +1,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
-
+    <uses-permission android:name="android.permission.INTERNET" />
```
They are granted here on the package itself:
https://github.com/UseAlloy/alloy-codeless-lite-android/blob/main/sdk/src/main/AndroidManifest.xml

Did you have to add them to run this example? I don't have it and runs on my android phone.

Austinj0 replied:
Oh I did not realize they were in the package itself. Hayman brought it up, let me confirm with him if there was a client issue that brought this up.

mrestuccia reviewed (approved):
I am OK with this. But would like to know why is needed.

Austinj0 committed:
Update AndroidManifest.xml

Reverting changes, as they are not needed because they are included in the codeless SDK package.

Austinj0 committed:
Update build.gradle to use 1.0.15 of the codeless SDK

Austinj0 commented:
@mrestuccia @borisonr  I reverted my changes to the manifest. I tested and confirmed locally it's not needed, it was just a misunderstanding of permissions. 
I updated build.gradle to use 1.0.15 of the SDK instead of the 1.0.9. So isNext wont be a required parameter. 
I couldn't see anywhere that Android Studio supported a @latest for dependencies.
